addons/mail/__manifest__.py

```diff
@@ -2,7 +2,7 @@
 
 {
     'name': 'Discuss',
-    'version': '1.6',
+    'version': '1.7',
```

Comment:
can't you remove `required=True` on both res_id field ?
(not necessary to keep 2 different SQL constraint for the same thing)

Reply:
Well it would be strange not to have it at model level ... unless this is annoying I would prefer keeping both.

Reply:
hum, is like having `index=True` on a field with a unique constraint (you will have 2 SQL constraints), those 2 fields are not visible in any view so for me it's ok to remove the `required=True`.

Let it be like this and let see awa opinion during the review

Reply:
Remove required.